engines/tika/src/main/java/org/alfresco/transform/tika/transformers/Tika.java

```diff
@@ -259,6 +259,7 @@ private ContentHandler getContentHandler(String targetMimetype, Writer output)
                 TransformerHandler transformerHandler;
                 transformerHandler = factory.newTransformerHandler();
                 transformerHandler.getTransformer().setOutputProperty(OutputKeys.INDENT, "yes");
+                transformerHandler.getTransformer().setOutputProperty(OutputKeys.VERSION, "1.1");
```

Comment:
That change will affect current XML contract. Please prepare the solution based on this comment: https://hyland.atlassian.net/browse/ACS-10404?focusedCommentId=2585083